docs/xgc.md

```diff
@@ -164,6 +167,25 @@ If these requirements are not met, the option is ignored.
 
 : Sets the initial scavenger tenure age in the generational concurrent GC policy. For more information, see [Tenure age](https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.vm.80.doc/docs/mm_gc_generational_tenure.html).
 
+### `tlhIncrementSize`
+
+        -Xgc:tlhIncrementSize=<bytes>
+
+: Sets the increment size of the thread local heap (TLH), which plays a key role in cache allocation. Threads start creating TLHs with a predefined initial size (default 2 KB). On every TLH refresh, the requested size for that thread is increased by an increment (default 4 KB). Use this option to control the increment size.
```

Comment:
Ah well, it's sort of described here. So it's up to you if it's appropriate to go to the details in at the first spot

Reply:
If your ok with the update here I think I prefer to leave it in the detail. Save cluttering the initial table. Thanks for your review. :+1: